### PR TITLE
Add GenJyuuGothic-l

### PR DESCRIPTION
**Pull request by tomy0000000**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask-fonts/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-fonts/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by tomy0000000** [comment]
I've run the following two checks: 
* `brew cask audit --download {{cask_file}}`
* `brew cask audit --new-cask {{cask_file}}`

which generate the same output indicate for adding `desc` in the cask.
Since I was just modifying from another font of this series, which also does not have a desc field.
Furthermore, after scrolling though some of the other casks, most of them didn't filled it, eihter.

Another note worth mention is that the host of this font seems to have very low bandwidth for downloading, I'm suspecting that was the reason why CI timed out, but cask is fully functioning in my local environment.

File: Casks/font-genjyuugothic-l.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+cask "font-genjyuugothic-l" do
+  version "20150607,8.643"
+  sha256 "d2fccec290232df110d1116fef4411416875acf7647084c9ab7d7eb5e8a80c50"
+
+  # osdn.jp/ was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/font-genjyuugothic-l.rb:5]: This isn’t enough disambiguation.

File: Casks/font-genjyuugothic-l.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  url "https://osdn.dl.osdn.jp/users/#{version.after_comma.major}/#{version.after_comma.no_dots}/genjyuugothic-l-#{version.before_comma}.zip"
```
Comment (vitorgalvao) [inline Casks/font-genjyuugothic-l.rb:6]: `osdn` URLs need to [follow a pattern](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
> Furthermore, after scrolling though some of the other casks, most of them didn't filled it, eihter.

Because `desc` is a recent addition and adding it is an ongoing process. All new casks must add it.

File: Casks/font-genjyuugothic-l.rb (hunk 1 above)
Comment (tomy0000000) [reply Casks/font-genjyuugothic-l.rb:5]: I'm not quite sure which part is ambiguous you mean by, could you give more suggestion or examples on how I should patch it?

File: Casks/font-genjyuugothic-l.rb (hunk 2 above)
Comment (tomy0000000) [reply Casks/font-genjyuugothic-l.rb:6]: This is the results of running `wget https://osdn.jp/downloads/users/8/8643/genjyuugothic-l-20150607.zip`
```
--2020-10-31 23:06:05--  https://osdn.jp/downloads/users/8/8643/genjyuugothic-l-20150607.zip
Resolving osdn.jp (osdn.jp)... 202.221.179.11
Connecting to osdn.jp (osdn.jp)|202.221.179.11|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://ja.osdn.net/downloads/users/8/8643/genjyuugothic-l-20150607.zip [following]
--2020-10-31 23:06:05--  https://ja.osdn.net/downloads/users/8/8643/genjyuugothic-l-20150607.zip
Resolving ja.osdn.net (ja.osdn.net)... 202.221.179.17
Connecting to ja.osdn.net (ja.osdn.net)|202.221.179.17|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://ja.osdn.net/frs/chamber_redir.php?m=gigenet&f=%2Fusers%2F8%2F8643%2Fgenjyuugothic-l-20150607.zip [following]
--2020-10-31 23:06:05--  https://ja.osdn.net/frs/chamber_redir.php?m=gigenet&f=%2Fusers%2F8%2F8643%2Fgenjyuugothic-l-20150607.zip
Reusing existing connection to ja.osdn.net:443.
HTTP request sent, awaiting response... 302 Found
Location: https://mirrors.gigenet.com/OSDN//users/8/8643/genjyuugothic-l-20150607.zip [following]
--2020-10-31 23:06:06--  https://mirrors.gigenet.com/OSDN//users/8/8643/genjyuugothic-l-20150607.zip
Resolving mirrors.gigenet.com (mirrors.gigenet.com)... 69.65.16.171
Connecting to mirrors.gigenet.com (mirrors.gigenet.com)|69.65.16.171|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 105857101 (101M) [application/zip]
```
I've tried all of the urls above but they all create `OSDN URL format incorrect.` error after running `brew cask audit --download font-genjyuugothic-l`. It seems like the developer of this font does not place this font in a dedicated project, but store it in a chamber of another font (as a varient, I guess?). I'm not quite familiar with OSDN and its environment, so correct me if I'm wrong. Any instruction or suggestion on how to resolve the correct url is appriciated.

File: Casks/font-genjyuugothic-l.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/font-genjyuugothic-l.rb:5]: `osdn.jp` is like using `github.com`, it doesn’t identify an author, and thus isn’t enough to claim the website is “official.” After changing the URL’s format, it should be easier.

File: Casks/font-genjyuugothic-l.rb (hunk 2 above)
Comment (vitorgalvao) [reply Casks/font-genjyuugothic-l.rb:6]: Leave as is for now, as the others have the same issue and we should deal with all at once.

**Review by vitorgalvao — APPROVED** [review]

**Comment by tomy0000000** [comment]
As I previously stated, CI timed out due to the low bandwidth of downloading, but I've run thruogh all the checks and they worked fined. Please review the latest changes, and reply if there's anything needs fixing.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**